projects/optic/src/commands/api/add.ts

```diff
@@ -121,6 +121,14 @@ async function promptForStandard(
 ): Promise<string | undefined> {
   const existingStandards = await client.getOrgStandards(orgId);
 
+  // if empty, create default without interrupting first time users
+  if (existingStandards.length === 0) {
+    const standard = await client.createOrgStandard(orgId, [
+      { name: 'breaking-changes', config: {} },
+    ]);
+    return standard.slug;
+  }
+
```

Comment:
Is this intended to be part of this PR?

If yes, should we add any logging here to show that a standard has been added?

Reply:
nope removing this